vector/src/main/java/im/vector/app/core/platform/VectorBaseActivity.kt

```diff
@@ -121,7 +120,6 @@ abstract class VectorBaseActivity<VB : ViewBinding> : AppCompatActivity(), Maver
 
     protected fun View.debouncedClicks(onClicked: () -> Unit) {
         clicks()
-                .throttleFirst(300)
```

Comment:
:100: this should help make the app feel a little snappier (if I've understood the current usages)

Reply:
No, it is not delaying the first click hopefully, but just ignore the next ones in a 300ms window